spec/system/reward_categories_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Reward categories', type: :system do
+  let!(:cat_rew1) { create(:category_reward) }
+  let!(:cat_rew2) { create(:category_reward) }
+  let!(:cat_rew3) { create(:category_reward) }
+  let!(:cat_rew4) { create(:category_reward) }
+  let!(:employee) { create(:employee) }
+
+  context 'when we want to check reward categories' do
```

Comment:
I would create a context + another context for category selected
```suggestion
  context 'when visiting rewards index page without a category selected' do
```